src/components/Link.astro

```diff
@@ -1,9 +1,13 @@
 ---
 const { href } = Astro.props;
+
+let target = href.startsWith('/') || href.startsWith('#')
```

Comment:
This logic has been pulled directly from our existing `ProseA.vue` component used by MDX files

Reply:
@tom-blake suggestion: Change `let` to `const`.